package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -114,6 +114,15 @@ protein, backbone, nucleic, nucleicbackbone
     is identfied by a hard-coded set of residue names so it  may not
     work for esoteric residues.
 
+water
+    selects all atoms that belong to a set of water residues; water
+    is defined with a set of common water abbreviations present in
+    topology files and may not work with certain water residue names.
+    Currently the following water resnames are supported:
+    3 letter resnames: ``H2O``, ``HOH``, ``OH2``, ``HHO``, ``OHH``, ``TIP``,
```

Comment:
I'm not sure this will render out amazingly well - unfortunately the docs build seems to have failed.